pandas/core/generic.py

```diff
@@ -4955,7 +4961,8 @@ def bfill(self, axis=None, inplace=False, limit=None, downcast=None):
 
         Returns
         -------
-        filled : %(klass)s
+        %(klass)s
+            Some values have been substituted for new values.
```

Comment:
Description could be improved here - perhaps "Object after replacement" or something similar